src/fetchData/index.ts

```diff
@@ -0,0 +1,17 @@
+import BaseApi from '../baseApi';
+import { DeltaSyncParams } from '../utils/params/deltaSyncParams';
+import { createRawApiResponse } from '../utils/response/rawApiResponse';
+import { FetchDataResponse } from './type';
+import { createFetchDataResponse } from './util';
+
+const resourceName = 'fetchData';
+
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+export class FetchDataEndpoint extends BaseApi<any> {
```

Comment:
Didn't have any choice but using `any` here. Seems very similar to implementation of `deltaSync` though.